### PR TITLE
Add setup detail to README

### PR DESCRIPTION
**Pull request by david90**

### CODE REVIEW

**Comment by oursky-travis** [comment]
@david90, thanks for your PR! By analyzing the annotation information on this pull request, we identified @royuen, @rickmak and @cheungpat to be potential reviewers

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
-This is a simple Raspberry Pi setup demonstrating the usage of Skygear JS SDK in
+# Skygear Working Example on Raspberry Pi
+
+This is a simple Raspberry Pi setup demonstrating the usage of [Skygear JS SDK](https://docs.skygear.io/js/guide/) in
 Raspberry Pi.
 
+This demo project demonstrates the following:
```
Comment (rickmak) [inline README.md]: followings

**Review by rickmak — CHANGES_REQUESTED** [review]
yay

File: README.md (hunk 1 above)
Comment (david90) [reply README.md]: updated

**Pull request merged**